spyder/plugins/editor/utils/folding.py

```diff
@@ -230,10 +233,22 @@ def detect_fold_level(self, prev_block, block):
             # ignore commented lines (could have arbitary indentation)
             prev_text = prev_block.text()
             indent_len = (len(prev_text) - len(prev_text.lstrip())) // prev_lvl
+            # Verify if the previous line ends with a continuation line
+            # with a regex
+            if (re.search(cont_line_regex, prev_block.text()) and
+               indent_len > prev_lvl):
```

Comment:
Please indent this line by 8 spaces from the `if` above it (currently it's only spaced by 3 spaces).